.github/workflows/lint.yml

```diff
@@ -11,16 +11,17 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 6
     steps:
-      - name: Set up Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: "1.16"
```

Comment:
`1.16` is the required version now for Starport. Should we rollback this change?

Reply:
The linter should be a prebuilt binary. You don't need to install go I believe

Reply:
Tested with https://github.com/tendermint/starport/pull/819, seems like removing version is not a problem indeed. While we were using the alpha version of `1.16`, it was somewhat required to manage Go installation ourselves because linter was not able to use non stable releases. I think the reason is the linter somehow uses `go` cmd under the hood or depends on a Go installation in some other way.